src/renderer/captureTool/js/capture.js

```diff
@@ -0,0 +1,1073 @@
+/**
+ * Capture Tool Renderer Window
+ *
+ * Copyright 2019 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+"use strict";
+(function (fluid) {
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.psp");
+    fluid.registerNamespace("gpii.captureTool");
+
+    // TODO Ultimately these should be relocated to a JSON5 file, probably in
+    // universal so they can be used by any application or reporting utility
+    // that needs to present the results to a human. This would all be a part
+    // of the "Presentation Registry" to be used across tools.
+    gpii.captureTool.appOrdering = [
+        "com.microsoft.windows.language",
+        "com.microsoft.windows.screenDPI",
+        "com.microsoft.windows.highContrast",
+        "com.microsoft.windows.brightness",
+        "com.microsoft.windows.nightScreen",
+        "com.microsoft.windows.cursors",
+        "com.microsoft.windows.mouseSettings",
+        "com.microsoft.windows.mouseTrailing",
+        "com.microsoft.windows.touchPadSettings",
+        "com.microsoft.windows.mouseKeys",
+        "com.microsoft.windows.filterKeys",
+        "com.microsoft.windows.stickyKeys",
+        "com.microsoft.windows.typingEnhancement",
+        "com.microsoft.windows.onscreenKeyboard",
+        "com.microsoft.windows.desktopBackground",
+        "com.microsoft.windows.desktopBackgroundColor",
+        "com.microsoft.windows.magnifier",
+        "com.microsoft.windows.narrator",
+        "com.texthelp.readWriteGold",
+        "com.office.windowsWordHome365LearningTools",
+        "com.office.windowsWordPro365LearningTools",
+        "com.office.windowsOneNoteLearningTools",
+        "com.freedomscientific.magic",
+        "org.nvda-project",
+        "com.freedomscientific.jaws",
+        "eu.singularlogic.pixelsense.sociable",
+        "net.gpii.uioPlus",
+        "com.microsoft.windows.screenResolution",
+        "net.opendirective.maavis",
+        "com.microsoft.windows.volumeControl",
+        "net.gpii.test.speechControl",
+        "net.gpii.explode"
```

Comment:
This one is not a solution that should be exposed to or mentioned to users

Reply:
removed these test GPII solutions